scripts/deploy_aws_gs_event_relay_user.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
I think this script is better named `create_aws_gs_event_relay_user.py` instead of `deploy_aws_gs_event_relay_user.py`. You also need to mention this script in the readme, somewhere alongside authorize_aws_deploy.

Reply:
will fix